converter/src/test/resources/cpc_plus/failure/CPCPlus_IncNumTotalMeasures_SampleQRDA-III.xml

```diff
@@ -92,7 +92,7 @@
 	  <participant typeCode="LOC">
       <associatedEntity classCode="SDLOC">
          <id root="2.16.840.1.113883.3.249.5.1"
-            extension="T2OR1234"
+            extension="DogCow"
```

Comment:
Moof!  Also, `TestApmEntityId` should work too, given that it is in `test_apm_entity_ids.json`.

Reply:
true, but I prefer the inter-species hybrid